docs/Changelog.md

```diff
@@ -8184,7 +8184,7 @@ This version of the operator has been available since version 9 of the default O
 
 <dl>
 <dt><tt>input</tt> : T1</dt>
-<dd>1D tensor. The shape of the expected output tensor. If empty tensor is given, the output would be a scalar.</dd>
+<dd>1D tensor. The shape of the expected output tensor. If empty tensor is given, the output would be a scalar. All values must be >= 0.</dd>
```

Comment:
Must be `>0`? I believe the `=` case corresponds to a scalar and we already have `empty tensor` for that.

Reply:
Yes, but we can a multidimensional tensor with one dim = 0.
Like:
numpy.full((2, 0), 4)
numpy.random.randn(2, 0)
or
torch.randn([0, 2, 3])

Reply:
Just to clarify: the output is expected to be an empty tensor, so the constant-value used is immaterial. Is that right?

Reply:
Yes, that's correct.